provision/internal/operator/terraform/gardener.go

```diff
@@ -55,6 +65,16 @@ func initGardenerProvider() error {
 		return err
 	}
 
+	if runtime.GOOS == "windows" {
```

Comment:
This block of code does the same as the one on line 33. Why duplicating it?

Reply:
I have fixed this issue

Reply:
It seems you wrote the function but are not using it, so this is not solved yet.